content/integrations/awsdynamo.md

```diff
@@ -4,6 +4,7 @@ integration_title: AWS DynamoDB
 kind: integration
 git_integration_title: amazon_dynamodb
 newhlevel: true
+description: "Track table size, read/write capacity, request latency, and more."
```

Comment:
"Track DynamoDB table size, read/write capacity, request latency, and more."

Reply:
👍